packages/cli/src/lib/packager/index.ts

```diff
@@ -89,6 +99,24 @@ export async function createDistWorkspace(
     const dest = typeof file === 'string' ? file : file.dest;
     await fs.copy(paths.resolveTargetRoot(src), resolvePath(targetDir, dest));
   }
+
+  if (options.skeleton) {
+    const skeletonFiles = targets.map(target => {
+      const dir = relativePath(paths.targetRoot, target.location);
+      return joinPath(dir, 'package.json');
+    });
+
+    await tar.create(
+      {
+        file: resolvePath(targetDir, options.skeleton),
+        cwd: targetDir,
+        portable: true,
+        noMtime: true,
+      } as CreateOptions & { noMtime: boolean },
```

Comment:
Heh, waddup with the unknown field

Reply:
not part of @types/tar for some reason, and wanted to get unblocked x)